app/views/layouts/_flashes.html.haml

```diff
@@ -6,8 +6,8 @@
         %p= alert
 
     - if notice
-      .ui.icon.warning.message
-        %i.warning.icon
+      .ui.icon.success.message
+        %i.success.icon
```

Comment:
:+1:

OK, but we do need positive / negative alerts distinct from info / warning. Let’s make another issue.